code/controllers/subsystem/ipintel.dm

```diff
@@ -0,0 +1,18 @@
+var/datum/subsystem/ipintel/SSipintel
+
+/datum/subsystem/ipintel
+	name = "IP Intel"
+	priority = -3
+	var/enabled = 0 //disable at round start to avoid checking reconnects
+	var/throttle = 0
+	var/errors = 0
+
+	var/list/cache = list()
+
+/datum/subsystem/ipintel/New()
+	NEW_SS_GLOBAL(SSipintel)
+
+/datum/subsystem/ipintel/Initialize(timeofday, zlevel)
+	enabled = 1
+	. = ..()
```

Comment:
So this subsystem is basically my OOP solution to the original hacky way i checked if the world had just started to avoid checking round start reconnects.

It has its failures, as it means there is a small window after world start where somebody could be a fresh connection, but never get checked, but at least its not more snowflake added to world/New()